css/style.css

```diff
@@ -0,0 +1,206 @@
+:root {
+  --baseGray-40: #b5bec6;
+  --baseWhite: #ffffff;
+  --baseGray-80: #4a5660;
+  --date-hover: #cbbcbc;
+}
+
+[data-theme="dark"] {
+  --baseGray-80: #ffffff;
+  --baseWhite: #464646;
+  --date-hover: #363232;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html,
+body {
+  height: 100%;
+  width: 100%;
+  background-color: var(--baseWhite);
+}
+
+.app {
+  height: 500px;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 0 auto;
+  border-radius: 8px;
+  justify-content: center;
+}
+
+.calendar {
+  background-color: var(--baseWhite);
+  display: inline-flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 22px;
+  padding: 24px;
+  border-radius: 8px;
+  overflow: hidden;
+  box-shadow: 0 0 20px 1px rgba(0, 0, 0, 0.1);
+}
+
+.display-month-year {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  align-self: stretch;
+}
+
+.icons {
+  width: 16px;
+  height: 16px;
+  border: none;
+  background-color: var(--baseWhite);
+  cursor: pointer;
+}
+
+.month-year-name {
+  width: fit-content;
+  margin-top: -1px;
```

Comment:
@raaj-oo7 Why this negative margin?

Reply:
@raaj-oo7 ???

Reply:
Display month year-name box is upper then previous and next button so i put negative margin...but this is minor difference... i remove negative margin and solve it